### PR TITLE
Create movie-quote-generator.yml

### PR DESCRIPTION
**Pull request by kamiri-charles**
This react project would be great for beginners in the open-source contribution, Thanks.

### CODE REVIEW

File: _data/projects/movie-quote-generator.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Movie Quote Generator
+desc: A simple web app that generates random movie quotes. Built with ReactJs.
+site: https://kamiri-charles.github.io/movie-quote-generator
+tags:
+  - reactjs
+  - webdesign
+  - javascript
+upforgrabs:
+  name: help wanted
+  link: https://github.com/kamiri-charles/movie-quote-generator/issues
```
Comment (ritwik12) [inline _data/projects/movie-quote-generator.yml]: Either change the name above or make this link as help wanted one @kamiri-charles

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/movie-quote-generator.yml (hunk 1 above)
Comment (kamiri-charles) [reply _data/projects/movie-quote-generator.yml]: Is this good??... @ritwik12

Comment (kamiri-charles) [reply _data/projects/movie-quote-generator.yml]: Hey, @ritwik12. I wanted the project to have a help wanted tag not up for grabs. I don't know how to change the link to be a help wanted one as the project is already labelled as help wanted. Please elaborate further.

Comment (shiftkey) [reply _data/projects/movie-quote-generator.yml]: Currently all of the open issues in this repository are not labelled:



If you wish to use the `help wanted` label simply update it to this when you have some issues labelled:

```suggestion
  link: https://github.com/kamiri-charles/movie-quote-generator/labels/help%20wanted
```

**Comment by ritwik12** [comment]
@kamiri-charles I see you updated the link. But your repo is still not having any issues marked with this label.
You can see that if anyone tries to open this link `https://github.com/kamiri-charles/movie-quote-generator/labels/help%20wanted` they will not be routed to your issues on your repo. So please go to your repo `https://github.com/kamiri-charles/movie-quote-generator/issues` and mark any or all issues here with this label.

**Comment by kamiri-charles** [comment]
Hey @ritwik12 , I have added "help wanted" labels to my issues as requseted.

**Review by ritwik12 — APPROVED** [review]

**Pull request merged**